Assets/MixedRealityToolkit-Tests/InteractionDefinitionTests.cs

```diff
@@ -439,5 +439,265 @@ public void Test14_TestSixDofNoChange()
         }
 
         #endregion SixDof
+
+        #region Interaction Array Tests
+
+        [Test]
+        public void Test15_InteractionArrayObject()
+        {
+            var interactions = new MixedRealityInteractionMapping[1];
+            interactions[0] = new MixedRealityInteractionMapping(1, AxisType.Raw, DeviceInputType.None, MixedRealityInputAction.None);
+            var testValue1 = (object)1f;
+            var testValue2 = (object)false;
+
+            var initialValue = interactions[0];
+
+            Assert.IsNotNull(initialValue);
+            Assert.IsNull(initialValue.GetRawValue());
+            Assert.IsFalse(initialValue.Changed);
+
+            initialValue.SetRawValue(testValue1);
+
+            Assert.IsTrue(initialValue.Changed);
+
+            var setValue1 = interactions[0];
+
+            Assert.IsNotNull(setValue1);
+            Assert.IsNotNull(setValue1.GetRawValue());
+            Assert.AreEqual(setValue1.GetRawValue(), testValue1);
+            Assert.IsFalse(setValue1.Changed);
+
+            setValue1.SetRawValue(testValue2);
+
+            Assert.IsTrue(setValue1.Changed);
+
+            var setValue2 = interactions[0];
+
+            Assert.IsNotNull(setValue2);
+            Assert.IsNotNull(setValue2.GetRawValue());
+            Assert.AreEqual(setValue2.GetRawValue(), testValue2);
+            Assert.IsFalse(setValue2.Changed);
+        }
+
+        [Test]
+        public void Test16_InteractionArrayBool()
+        {
+            var interactions = new MixedRealityInteractionMapping[1];
+            interactions[0] = new MixedRealityInteractionMapping(1, AxisType.Digital, DeviceInputType.None, MixedRealityInputAction.None);
+            var testValue1 = true;
+            var testValue2 = false;
+
+            var initialValue = interactions[0];
+
+            Assert.IsNotNull(initialValue);
+            Assert.IsFalse(initialValue.GetBooleanValue());
+            Assert.IsFalse(initialValue.Changed);
+
+            initialValue.SetBoolValue(testValue1);
+
+            Assert.IsTrue(initialValue.Changed);
+
+            var setValue1 = interactions[0];
+
+            Assert.IsNotNull(setValue1);
+            Assert.IsTrue(setValue1.GetBooleanValue());
```

Comment:
Can we standardise on the Bool get / set.
Currently it is:
* GetBooleanValue
* SetBoolValue

Reply:
Def. I might update that when I rename SixDof to MixedRealityPose